api/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/SUSE/metabroker/api
+
+go 1.14
+
+require (
+	k8s.io/api v0.17.2
+	k8s.io/apimachinery v0.17.2
+	sigs.k8s.io/controller-runtime v0.5.0
+)
```

Comment:
Probably needs a new `doc.go` (or `README.md`, depending on what you think fits better).

Reply:
A `doc.go` sounds more appropriate.

Reply:
Done in https://github.com/SUSE/metabroker/pull/22/commits/717e9f45f3be2dda7dde33854b67d020cba58718.